contracts/interfaces/IDCAStrategies.sol

```diff
@@ -412,6 +425,12 @@ interface IDCAStrategiesPositionsHandler {
     IDCAStrategies.PermissionSet[] permissions;
   }
 
+  // NOTE: i'm doing it this way because of a strange solidity error
+  struct TokenAmounts {
```

Comment:
What error where you getting (we can sync offline if you want)? Because we should be returning an array of a `struct { address token; uint256 amount }`

Reply:
It's strange but is working right now. So... fixed? 😄

Reply:
Hahaha 🤷

Reply:
Hahaha 🤷